src/Command/CreateControllerCommand.php

```diff
@@ -39,11 +39,13 @@ public function configure() {
      */
     public function execute(InputInterface $input, OutputInterface $output) {
 
+            $className = $input->getArgument('name');
+
             $name = $input->getArgument('name'). ".php";
```

Comment:
You can reuse variable from the line above. On top of that, name `$name` is very vague. 

How about this?
```suggestion
            $fileName = $className. '.php';
```

Notice that I also changed double quotes to single ones, which is also should be your default choice for any strings.